src/core/settings/settings.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { SettingsEntity } from './settings.model';
+
+interface SettingsContext {
+  userSettings: SettingsEntity;
+  setUserSettings: (settings: SettingsEntity) => void;
+}
+
+export const settingsContext = React.createContext<SettingsContext>({
+  userSettings: {
+    numberQuestions: 15,
```

Comment:
Let's move this magic number to a const

const defaultNumberOfQuestions = 15;

Maybe we can move to to core const

Reply:
Move the secondChanceEnabled to a const, defaulSecondChangeEnabled and set it to true by default